src/main/java/com/iexec/sms/secret/web2/Web2SecretService.java

```diff
@@ -38,7 +38,7 @@ protected Web2SecretService(EncryptionService encryptionService,
     }
 
     public Optional<Web2Secret> getSecret(String ownerAddress, String secretAddress) {
-        return web2SecretRepository.findByOwnerAddressAndAddressAllIgnoreCase(ownerAddress, secretAddress);
+        return web2SecretRepository.find(ownerAddress, secretAddress);
```

Comment:
```suggestion
        return web2SecretRepository.findById(new Web2SecretHeader(ownerAddress, secretAddress));
```
(With that you can remove default methods from interfaces)

Reply:
OK, please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/140/commits/a16d5c847f93e6ef6e0f3e68015849527106a6af.

Reply:
Excellent :pray: